circuits/package.json

```diff
@@ -19,7 +19,7 @@
     "compile": "circom circuits/main.circom --r1cs --sym --wasm -o zk/ ",
     "setup:sample-zkey": "snarkjs groth16 setup zk/main.r1cs  powersOfTau28_hez_final_15.ptau zk/main.zkey",
     "contribute:deterministic": "snarkjs zkey contribute ./zk/main.zkey ./zk/main_final.zkey --name='contributer_1' -v -e='00000'",
-    "export:verifier": "snarkjs zkey export solidityverifier zk/main.zkey zk/main_verifier.sol",
+    "export:verifier": "snarkjs zkey export solidityverifier zk/main_final.zkey zk/main_verifier.sol",
```

Comment:
You could output this directly into `contracts/src/ZKubeVerifier.sol` from the script

Reply:
I would prefer to have a separate script to copy verifier contract to `contracts` directory, since `build:dev` script calls `export:verifier`. This way we keep the workspaces separate if we explicitly don't want the opposite. The same goes for `compile` script. 
I suggest we rename `export:verifier` to `generate:verifier` and to have a separate script named `export:elements` which copies `main.wasm` and `main_final.zkey` to `app/public/zk` and `main_verifier.sol` to `contracts/src/ZKubeVerifier.sol`.
Also, none of the examples I followed outputs directly to `contracts` or `app` directory.

Reply:
Also, renaming `main.circom` to `zkube.circom`

Reply:
That sounds good to me